_includes/page-core/page-header.html

```diff
@@ -35,7 +35,10 @@
                                 <a href="/docs/networking">Networking</a>
                             </li>
                             <li>
-                                <a href="/docs/driver-overview">Hardware Drivers</a>
+                                <a href="http://docs.ev3dev.org/projects/lego-linux-drivers/en/ev3dev-jessie/">
+                                    Hardware Drivers
+                                    <i class="fa fa-external-link" id="external-link-nav-icon" aria-hidden="true"></i>
```

Comment:
Aren't ids supposed to be unique per page? It seems like you want a class rather than an id here.